docusaurus/docs/easythreads/panels/_category_.json

```
@@ -0,0 +1,8 @@
+{
+    "label": "Panels",
+    "position": 5,
```

```suggestion
    "position": 4,
```
Premium Guide and Best Practices should be the two final / last categories.